roles/mysql/tasks/install.yml

```diff
@@ -5,19 +5,21 @@
 # - 7 DB config
 # - 2 record as installed
 
-- name: 'Install MySQL packages: mariadb-server, mariadb-client, and 9 php packages (debuntu)'
+- name: 'Install MySQL packages: mariadb-server, mariadb-client, and php packages (debuntu)'
   package:
     name:
       - mariadb-server
       - mariadb-client
       # - php{{ php_version }}    # On Ubuntu 20.04 (and prob other OS's) this forces the install of: apache2 apache2-bin apache2-data apache2-utils libapache2-mod-php{{ php_version }} ETC
       - php{{ php_version }}-common    # 2020-05-21: @jvonau suggests this to avoid Apache above.  Or its superset php{{ php_version }}-cli if absolutely nec?
+      - php{{ php_version }}-curl
       - php{{ php_version }}-mysql
       - php-pear
       - php{{ php_version }}-gd
       - php{{ php_version }}-imap
       - php{{ php_version }}-ldap
       - php{{ php_version }}-odbc
+      - php{{ php_version }}-sqlite3
```

Comment:
@jvonau is it definitive that (mandatory) mysql requires sqlite3 ?

Or can we try removing this line?